pkg/admission/attributes.go

```diff
@@ -21,14 +21,16 @@ import (
 )
 
 type attributesRecord struct {
+	kind      string
 	namespace string
 	resource  string
 	operation string
 	object    runtime.Object
 }
 
-func NewAttributesRecord(object runtime.Object, namespace, resource, operation string) Attributes {
+func NewAttributesRecord(object runtime.Object, kind, namespace, resource, operation string) Attributes {
```

Comment:
kind is deducible from resource and vs versa-- do you really need to make everyone pass both?